### ISSUE TITLE
while playing with recommender widget, noticed that underscore must be loaded globally - which is weird

### ISSUE THREAD

**Issue by romanchyla** [adsabs/bumblebee#2074]
### Expected Behavior

'underscore' is normally imported by modules

### Actual Behavior

https://github.com/adsabs/bumblebee/blob/master/src/js/widgets/facet/widget.js has no such import

yet `_.debounce()` calls probably succeed

Suggests '_' is a global variable. Which it (normally) shouldn't be (conflicts)

**Comment by thostetler** [comment]
Yeah its always been a global, lodash/underscore add it to the global namespace by default.  We don't explicitly add a shim for this in discovery.config.  We could do that, but there are a number of places expecting a global, just like with Backbone and a few others -- I have them ignored by the linter here: https://github.com/adsabs/bumblebee/blob/master/.eslintrc.json
If we add in the shim, we'll have to refactor some to include '_' in the deps (preferred, but might be a pain).

**Comment by romanchyla** [comment]
Is webpack adding it to globals? I'm sure it wasn't always global (unless
lodash changed something dramatical)
